warning: absolute value function 'abs' given an argument of type 'long long' but has parameter of type 'int' which may cause truncation of value

Index: ROX-Filer/src/type.c
--- ROX-Filer/src/type.c.orig
+++ ROX-Filer/src/type.c
@@ -466,7 +466,7 @@ MaskedPixmap *type_to_icon(MIME_type *type)
 	if (type->image)
 	{
 		/* Yes - don't recheck too often */
-		if (abs(now - type->image_time) < 2)
+		if (llabs(now - type->image_time) < 2)
 		{
 			g_object_ref(type->image);
 			return type->image;
